src/sqlancer/timescaledb/gen/TimeScaleDBCommon.java

```diff
@@ -0,0 +1,17 @@
+package sqlancer.timescaledb.gen;
+
+import sqlancer.common.query.ExpectedErrors;
+
+public final class TimeScaleDBCommon {
+    private TimeScaleDBCommon() {
+
+    }
+
+    public static void addTimeScaleDBErrors(ExpectedErrors errors) {
+        errors.add("Error updating TimescaleDB when using a third-party PostgreSQL administration tool");
+        errors.add("Log error: could not access file \"timescaledb\"");
+        errors.add("ERROR: could not access file \"timescaledb-\\<version>\": No such file or directory");
```

Comment:
Errors should be specific to certain kinds of operations (e.g., updating data or evaluating expressions). When do errors like "could not access file" happen? Is this something we can prevent?

Reply:
Oh, I see. But if the user does not have the right to access certification resources, this is not an expected error?

Reply:
Could you give a bit more information on when this could happen? In general, we should assume that a specific setup of the database system is required (including sufficient permissions), to efficiently test the system.

Reply:
Acutally, I copied the errors from the doc of TimescaleDB: https://docs.timescale.com/timescaledb/latest/how-to-guides/troubleshoot-timescaledb/

Reply:
Okay, I see. I think we can remove these errors. Such errors should be added only when we cannot prevent them with reasonable effort. See https://github.com/sqlancer/sqlancer/blob/master/CONTRIBUTING.md#expected-errors.